olive/strategy/search_strategy.py

```diff
@@ -213,6 +213,7 @@ def _create_searcher(self, search_space_names: List[str]) -> SearchAlgorithm:
             searcher = REGISTRY[self._config.search_algorithm](
                 search_spaces_dict, objectives, higher_is_betters, self._config.search_algorithm_config
             )
+            searcher.initialize()
```

Comment:
This line is the fixing for ``__init__` method calls overridden method`, right?

Reply:
yeah